Tests/UIKitTests/UIViewControllerExtensionsTests.swift

```diff
@@ -131,6 +131,17 @@ final class UIViewControllerExtensionsTests: XCTestCase {
 
         XCTAssertNil(childViewController.parent)
     }
+    
+    func testSetTabBarImage() {
+        let viewController = UIViewController()
+        let imageName = "sun.max"
+        let title = "Sun"
+        viewController.setTabBarImage(SFImageName: imageName, title: title)
+        let testImage = UIImage(systemName: imageName, withConfiguration: UIImage.SymbolConfiguration(scale: .large))
+        
+        XCTAssertEqual(viewController.tabBarItem.image?.isSymbolImage, testImage?.isSymbolImage)
```

Comment:
Does this pass? ViewController is not in a TabBarController ... so I'm confused. The first check shouldn't pass IIUC

Reply:
Also, should we return a bool indicating it was set instead of assert?
We could also add test cases for the two error paths =]

Reply:
Thank you for the reply I will fix it and improve it for the next submission. : )